model/trace/database.yaml

```diff
@@ -5,18 +5,7 @@ groups:
     brief: This group defines the attributes used to perform database client calls.
     attributes:
       - ref: db.query.text
-        requirement_level:
-          recommended: >
-            Non-parameterized query text SHOULD NOT be collected by default unless there is sanitization that excludes
-            sensitive data, e.g. by redacting all literal values present in the query text.
-
-            Parameterized query text SHOULD be collected by default
-            (the query parameter values themselves are opt-in,
-            see [`db.query.parameter.<key>`](../../docs/attributes-registry/db.md)).
-        note:
-          Even though parameterized query text can potentially have sensitive data, by using a parameterized query
-          the user is giving a strong signal that any sensitive data will be passed as parameter values, and the benefit
-          to observability of capturing the static part of the query text by default outweighs the risk.
```

Comment:
I'm not sure this was intentionally defined here in `attributes.db.client.minimal`. This was overwritten below in `trace.db.common.query_and_collection` (which extends `attributes.db.client.minimal`), which resulted in this text never being rendered in any of the `.md` files. Some specific db specs extend `attributes.db.client.minimal`, but none of them seem to use `db.query.text`.

So I moved the whole text below to `trace.db.common.query_and_collection`, which seem to make more sense (e.g. that way `database-spans.md` is correct, but may not be ok for some other specs).

I think this should be cleaned up - let's discuss in the WG meeting.

Reply:
Should be ok now.